fairscale/optim/oss.py

```diff
@@ -160,15 +159,6 @@ def local_params(self) -> List[torch.Tensor]:
         # Make sure that the iterator is not consumed, only expose a copy
         return self._local_params
 
-    @property
-    def index_to_param(self) -> Dict[int, torch.Tensor]:
```

Comment:
this was an optimisation for very big models, which I think was a mistake: this increased the code footprint, and really, really big models will not be trained with OSS anyway. On top of that, this is not needed with this PR except for torch 1.5, which will be deprecated soonish